recipes/pyramids/meta.yaml

```diff
@@ -0,0 +1,58 @@
+{% set name = "pyramids" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/MAfarrag/pyramids/archive/{{ version }}.tar.gz
+  sha256: 7759146e4642f84c87c1996bfc21a72209e0bd0e994a521a2bce3cddbceff7a3
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
```

Comment:
You more or less need to copy entry points:
https://github.com/MAfarrag/pyramids/blob/main/setup.py#L32

For example:
https://github.com/conda-forge/pytest-feedstock/blob/main/recipe/meta.yaml#L16

Reply:
I don't understand why you makred this as resolved, even though you did not add any entry points.